libraries/rush-lib/src/logic/operations/ShellOperationRunnerPlugin.ts

```diff
@@ -91,7 +99,7 @@ export function initializeShellOperationRunner(options: {
   }
 
   if (rawCommandToRun) {
-    const { rushConfiguration, commandForHash: rawCommandForHash } = options;
+    const { commandForHash: rawCommandForHash } = options;
```

Comment:
Nit: Get the environment here instead of referring to `options.environment` below